packages/camlimages/camlimages.5.0.3/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
```

Comment:
This needs to be upgraded to opam 2.0 format.  This can be done by `opam package-upgrade <opam file>` and then run `opam lint` to correct any errors.

Reply:
done!